Source/ARTDeviceDetails.m

```diff
@@ -0,0 +1,41 @@
+//
+//  ARTDeviceDetails.m
+//  Ably
+//
+//  Created by Ricardo Pereira on 07/02/2017.
+//  Copyright © 2017 Ably. All rights reserved.
+//
+
+#import "ARTDeviceDetails.h"
+#import "ARTDevicePushDetails.h"
+
+NSString *const ARTDevicePlatform = @"ios";
+NSString *const ARTDeviceFormFactor = @"mobile";
```

Comment:
Really hard-coded const?  Given this could run on tablet or watch, I don't think this is right.  Equally, we should really support MacOS at some point too.